app/services/save_and_send_reject_by_default_feedback.rb

```diff
@@ -14,5 +14,16 @@ def call!
 
       CandidateMailer.feedback_received_for_application_rejected_by_default(application_choice).deliver_later
     end
+
+    notify_slack
+  end
+
+  def notify_slack
+    provider_name = application_choice.offered_course.provider.name
+    candidate_name = application_choice.application_form.first_name
+    message = ":telephone_receiver: #{provider_name} has just sent feedback for #{candidate_name}’s RBD application"
```

Comment:
```suggestion
    message = ":telephone_receiver: #{provider_name} has sent feedback for #{candidate_name}’s RBD application"
```

As per @duncanjbrown in https://github.com/DFE-Digital/apply-for-teacher-training/pull/3420

Reply:
Haha, I'm _just_ so conditioned to expect 'just' in Slack messages after all this time.